formats/coming_soon/publisher/links.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "replaced_content_item"
+  ],
+  "properties": {
+    "replaced_content_item": {
```

Comment:
I agree, though it's less that we want consistency so much as we programmatically need it.  Two options: 1) this is a temporary format, or 2) can you think of a name that's more flexible?